### ISSUE TITLE
Some Cyrillic glyphs are not available in Cambria

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#634]
From https://github.com/metanorma/iso-9/issues/9 by @Intelligent2013 :

From mn2pdf log:
```
Glyph "?" (0x521) not available in font "Cambria".
Glyph "?" (0x523) not available in font "Cambria".
```
These glyphs renders as `#` (line 48 and 55):
![image](https://user-images.githubusercontent.com/5559926/165050845-6972c4e3-e60d-46fd-a558-643536aa7eeb.png)

Adoc:
```adoc
| 46 | &#x43B; | l
| 47 | &#x43B;&#795; | &#x13A;
| 48 | &#x521; | &#x13C;
. . .
| 52 | &#x43D;&#807; | n&#803;
| 53 | &#x4A5; | n&#856;
| 54 | &#x43D;&#808; | n&#768;
| 55 | &#x523; | &#x144;
| 56 | &#x4CA; | &#x148;
```

These are "El" and "En" "with middle hook":
* https://en.wikipedia.org/wiki/El_with_middle_hook
* https://en.wikipedia.org/wiki/En_with_middle_hook

Cambria doesn't have them.

**Comment by ronaldtse** [comment]
Cambria (even the latest version) does not have the 513 and 523 points:
![image](https://user-images.githubusercontent.com/11865/165056144-3a5c3f70-a14d-4f8a-badb-80aa05d9505e.png)

Times New Roman from 2017 has them:
![image](https://user-images.githubusercontent.com/11865/165056195-3f0b8e69-f427-432f-aad6-0f8410475b74.png)

Adobe Source Serif does not have them:
![image](https://user-images.githubusercontent.com/11865/165056271-11323022-ee92-4128-95cb-42cd30a8637d.png)

Noto Sans has all of them:
![image](https://user-images.githubusercontent.com/11865/165056309-3b2b47bd-faba-449b-bc24-0cd8c067c4a1.png)

I suggest that we consider switching from Adobe Source as our backstop to using Noto as our backstop font.

**Comment by ronaldtse** [comment]
Once https://github.com/metanorma/mn2pdf/issues/119 is done I expect this problem to go away.

**Comment by ronaldtse** [comment]
In Office 2022 on Mac (Office 365), the Cambria version is 6.90i (2015). It does not contain these glyphs (521, 523).
<img width="785" alt="Screenshot 2022-04-27 at 8 28 53 PM" src="https://user-images.githubusercontent.com/11865/165528357-a46aec5b-8375-4c25-880a-2336aee8d5c8.png">

In Windows 11, it uses Cambria version 6.99. @Intelligent2013 found a version 6.98 (2017) of Cambria online, and it does contain these glyphs.
<img width="781" alt="Screenshot 2022-04-27 at 9 16 48 PM" src="https://user-images.githubusercontent.com/11865/165529048-12b74498-256a-4256-abef-57e33a314b24.png">

So this is an issue on non-Windows platforms. In any case, Noto is a good replacement.

**Comment by Intelligent2013** [comment]
![image](https://user-images.githubusercontent.com/5559926/172192896-a27ecca3-d367-4a0d-80e9-bb3846c9dca4.png)

**Comment by ronaldtse** [comment]
Thank you @Intelligent2013 !

**Comment by Intelligent2013** [comment]
Missing PDF Issue fixed in mn2pdf v1.46.

**Comment by ronaldtse** [comment]
Confirm that 48 and 55 now show properly in PDF.

<img width="666" alt="Screenshot 2022-06-08 at 9 11 22 AM" src="https://user-images.githubusercontent.com/11865/172504592-01673d0d-293f-4f39-864a-704956ba1d5b.png">

And that these glyphs are in Noto Serif while the others remain in Cambria.
<img width="885" alt="Screenshot 2022-06-08 at 9 12 52 AM" src="https://user-images.githubusercontent.com/11865/172504704-dd6c3766-693b-420f-896b-0b0691d2dd13.png">

Thanks!
